src/BuildScriptGeneratorCli/Commands/OptionTemplates.cs

```diff
@@ -9,26 +9,58 @@ namespace Microsoft.Oryx.BuildScriptGeneratorCli
 {
     internal static class OptionTemplates
     {
-        public const string Platform = "--platform <name>";
-        public const string PlatformVersion = "--platform-version <version>";
-
-        public const string RuntimePlatform = "--runtime-platform <name>";
-        public const string RuntimePlatformVersion = "--runtime-platform-version <version>";
+        public const string Log = "--log-file";
+        public const string LogDescription = "The file to which the log will be written.";
+        public const string Debug = "--debug";
+        public const string DebugDescription = "Print stack traces for exceptions.";
+        public const string Platform = "--platform";
+        public const string PlatformDescription = "The name of the programming platform used in the provided source directory.";
+        public const string PlatformVersion = "--platform-version";
+        public const string PlatformVersionDescription = "The version of the programming platform used in the provided source directory.";
 
         [Obsolete("Previous name for --platform")]
-        public const string Language = "-l|--language <name>";
+        public static readonly string[] Language = new string[] { "-l", "--language" };
+        public static readonly string LanguageDescription = "The name of the programming platform used in the provided source directory.";
         [Obsolete("Previous name for --platform-version")]
-        public const string LanguageVersion = "--language-version <version>";
-
-        public const string Property = "-p|--property <key-value>";
-        public const string ManifestDir = "--manifest-dir <directory-path>";
-        public const string BuildCommandsFileName = "--buildcommands-file <file-name>";
-        public const string AppType = "--apptype <application-type>";
-        public const string CompressDestinationDir = "--compress-destination-dir";
-        public const string DynamicInstallRootDir = "--dynamic-install-root-dir <dir-path>";
-
-        public const string EventName = "--event-name <event-name>";
-        public const string ProcessingTime = "--processing-time <processing-time>";
-        public const string CallerId = "--caller-id <caller-id>";
+        public static readonly string LanguageVersion = "--language-version";
```

Comment:
Nit: Is there a reason why some of these `string` fields are const, and some are static readonly? Lets try to opt for consistency wherever we can